_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_cenc.cpp

```diff
@@ -129,6 +129,9 @@ namespace UMC_HEVC_DECODER
         for(; index < rps->getNumberOfNegativePictures() + rps->getNumberOfPositivePictures(); index++)
                 pocList[numRefPicSetStCurrBefore + numRefPicSetStCurrAfter++] = picParam->CurrPic.pic_order_cnt + rps->getDeltaPOC(index);
 
+        if(!(supplier->GetDPBList()))
+            throw h265_exception(UMC_ERR_NULL_PTR);
```

Comment:
`UMC_ERR_NULL_PTR` is described as
> null pointer in input parameters
at `_studio\shared\umc\core\umc\include\umc_structures.h`. So, I'd rather use `UMC_ERR_FAILED` instead.

Reply:
Use UMC_ERR_FAILED replacing the UMC_ERR_NULL_PTR